code/modules/projectiles/guns/pistols.dm

```diff
@@ -728,3 +728,40 @@ It is a modified Beretta 93R, and can fire three round burst or single fire. Whe
 	scatter_unwielded = 5
 	recoil = -2
 	recoil_unwielded = -2
+
+/obj/item/weapon/gun/pistol/boltpistol
+	name = "\improper Bolt pistol"
+	desc = "Powerful and utilitarian side arms, Bolt Pistols have seen service within many of the Imperium's military forces since its inception in the 30th Millennium."
```

Comment:
```suggestion
	desc = "Powerful and utilitarian side arms, bolt pistols have seen service within many of the Imperium's military forces since its inception in the 30th millennium."
```